### ISSUE TITLE
Feature request: Lookup bar

### ISSUE THREAD

**Issue by SaltfishAmi** [birtles/rikaichamp#285]
There is a lookup bar feature in Rikaichan, like this:

![pic: lookupbar in rikaichan](https://user-images.githubusercontent.com/14184974/83426231-6dd9e180-a461-11ea-8b80-68c8fcc21318.png)

This is very convenient and I hope it could be in Rikaichamp!
otherwise I'll need to open a random pastebin and type there :(

**Comment by birtles** [comment]
Yes, I often find myself loading up any random website with a text field just so I can type something in and use Rikaichamp on it!

**Comment by Lebon14** [comment]
YES PLEASE.
